config/rootfs_overlay/root/setup.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
What's wrong with bash?

Reply:
It wasn't working on buildroot for me but /bin/sh was. have you been able to get it to work with the bash shebang?

Reply:
I haven't tried. But since you mention this is executed in the buildroot environment, it makes sense since buildroot only has busybox which emulates sh and probably doesn't emulate bash (I didn't verify).